src/i18n/locales/ko.json

```
@@ -15,7 +15,7 @@
     "light": "밝은",
     "dark": "어두운",
     "bots": "AI 봇",
-    "showMenuBar": "메뉴 바 표시"
+    "showMenuBar": "메뉴 표시줄 표시"
   },
   "common": {
     "apiKey": "API 키"
```

```suggestion
    "apiKey": "API 키",
```
A comma is required. I don't have permission to push to your repo. So you must modify it by your self.